lisp/bazel-mode.el

```diff
@@ -128,6 +128,10 @@
 ;;;###autoload
 (define-derived-mode bazel-mode prog-mode "Bazel"
   "Major mode for editing Bazel BUILD and WORKSPACE files."
+  ;; Almost all Starlark code in existence uses 4-character indentation and no
```

Comment:
fyi, that's mandated by Buildifier
(and tabs are forbidden in Starlark)

Reply:
thanks, reworded a bit